src/train_model.py

```diff
@@ -81,9 +86,11 @@ def train_model(config: Struct):
     if config.model_type.lower() == "longnet":
         model = LongNetModel(config)
```

Comment:
Same as previous comments about `LongNetLightning`.

Reply:
This issus is fixed.